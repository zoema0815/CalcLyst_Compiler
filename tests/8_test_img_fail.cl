int main () {
    Real a;
    Imaginary c;
    a = 3.7r;
    c = a;
    return 0;
}