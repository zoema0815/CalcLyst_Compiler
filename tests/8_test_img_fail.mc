int main () {
    Real a;
    Imaginary b;
    Imaginary c;
    Complex x;
    a = 3.7r;
    b = 3.2i;
    c = a;
    return 0;
}