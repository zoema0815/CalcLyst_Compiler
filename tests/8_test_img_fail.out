Fatal error: exception Failure("illegal assignment Imaginary = Real in c = a")
